### PR TITLE
Add support raw container in the map task

### PR DESCRIPTION
**Pull request by pingsutw**
# TL;DR
1. Add support raw container in the map task
2. Use default command of the raw container in the map task instead of replacing it with `pyflyte-map-execute ....`

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
image

## Tracking Issue
https://flyte-org.slack.com/archives/CP2HDHKE1/p1678230956906899

## Follow-up issue
https://github.com/flyteorg/flytecopilot/pull/54
https://github.com/flyteorg/flyteplugins/pull/329

### CODE REVIEW

**Review by wild-endeavor — DISMISSED** [review]

**Comment by wild-endeavor** [comment]
before merging, let's copy the minimum copilot and propeller versions into the PR description.

**Comment by kumare3** [comment]
@pingsutw shall we merge this?

**Comment by davidmirror-ops** [comment]
@pingsutw With the move to ArrayNode (not experimental anymore), should we close this?

**Comment by litaifang** [comment]
ArrayNode does not seem to support ContainerTask?

**Comment by eapolinario** [comment]
We'll need to port https://github.com/flyteorg/flyteplugins/pull/329 to arraynode.
